### ISSUE TITLE
After checking the price of an object, the application closes

### ISSUE THREAD

**Issue by Artick678** [SnosMe/awakened-poe-trade#1391]
after checking the price of an item, the app closes, it only lets me check one price for each time I open awakened

**Comment by elmo667** [comment]
Yep, same here. In the console it just says Segmentation Fault, so its probably a use after free, double free or something along those lines. As it only happens in 3.25 and theres been no update to awakened poe trade, its some edge case taking effect only now in the new league, possibly due to new modifiers or maybe different API responses

**Comment by Negatee** [comment]
same here

**Comment by markyzero** [comment]
same thing on Kubuntu 24.04.
Awakened-PoE-Trade-3.24.10003.AppImage --no-updates works well but both these versions segfault:
- Awakened-PoE-Trade-3.24.10003.AppImage
- Awakened-PoE-Trade-3.25.101.AppImage

running with sudo and --no-sandbox gives me the following:
`sudo ./Awakened-PoE-Trade-3.25.101.AppImage --no-sandbox
[844745:0729/163750.642526:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[844745:0729/163750.642623:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[844745:0729/163750.642636:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[844745:0729/163750.642649:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
Authorization required, but no authorization protocol specified

Authorization required, but no authorization protocol specified

Authorization required, but no authorization protocol specified

[844745:0729/163750.767368:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[844745:0729/163750.797120:ERROR:object_proxy.cc(576)] Failed to call method: org.freedesktop.DBus.NameHasOwner: object_path= /org/freedesktop/DBus: unknown error type: 
Authorization required, but no authorization protocol specified

[844745:0729/163750.946922:ERROR:network_service_instance_impl.cc(599)] Network service crashed, restarting service.
[844745:0729/163751.100929:ERROR:network_service_instance_impl.cc(599)] Network service crashed, restarting service.
[844745:0729/163751.252928:ERROR:network_service_instance_impl.cc(599)] Network service crashed, restarting service.
Authorization required, but no authorization protocol specified

Authorization required, but no authorization protocol specified

[845930:0729/163751.400232:ERROR:angle_platform_impl.cc(44)] DisplayVkXcb.cpp:58 (initialize): xcb_connect() failed, error 1
ERR: DisplayVkXcb.cpp:58 (initialize): xcb_connect() failed, error 1
[845930:0729/163751.400426:ERROR:angle_platform_impl.cc(44)] Display.cpp:1086 (initialize): ANGLE Display::initialize error 12289: Not initialized.
ERR: Display.cpp:1086 (initialize): ANGLE Display::initialize error 12289: Not initialized.
[845930:0729/163751.400513:ERROR:gl_display.cc(515)] EGL Driver message (Critical) eglInitialize: Not initialized.
[845930:0729/163751.400572:ERROR:gl_display.cc(786)] eglInitialize SwANGLE failed with error EGL_NOT_INITIALIZED
[845930:0729/163751.400653:ERROR:gl_display.cc(820)] Initialization of all EGL display types failed.
[845930:0729/163751.400729:ERROR:gl_ozone_egl.cc(26)] GLDisplayEGL::Initialize failed.
[844745:0729/163751.401911:ERROR:network_service_instance_impl.cc(599)] Network service crashed, restarting service.
Authorization required, but no authorization protocol specified

Authorization required, but no authorization protocol specified

[845930:0729/163751.524383:ERROR:angle_platform_impl.cc(44)] DisplayVkXcb.cpp:58 (initialize): xcb_connect() failed, error 1
ERR: DisplayVkXcb.cpp:58 (initialize): xcb_connect() failed, error 1
[845930:0729/163751.524476:ERROR:angle_platform_impl.cc(44)] Display.cpp:1086 (initialize): ANGLE Display::initialize error 12289: Not initialized.
ERR: Display.cpp:1086 (initialize): ANGLE Display::initialize error 12289: Not initialized.
[845930:0729/163751.524523:ERROR:gl_display.cc(515)] EGL Driver message (Critical) eglInitialize: Not initialized.
[845930:0729/163751.524562:ERROR:gl_display.cc(786)] eglInitialize SwANGLE failed with error EGL_NOT_INITIALIZED
[845930:0729/163751.524621:ERROR:gl_display.cc(820)] Initialization of all EGL display types failed.
[845930:0729/163751.524671:ERROR:gl_ozone_egl.cc(26)] GLDisplayEGL::Initialize failed.
[845930:0729/163751.529922:ERROR:viz_main_impl.cc(198)] Exiting GPU process due to errors during initialization
Authorization required, but no authorization protocol specified

Authorization required, but no authorization protocol specified

[844745:0729/163751.551064:ERROR:network_service_instance_impl.cc(599)] Network service crashed, restarting service.
[844745:0729/163751.696665:ERROR:network_service_instance_impl.cc(599)] Network service crashed, restarting service.
[844745:0729/163751.938308:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[844745:0729/163751.972073:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[846727:0729/163751.975703:ERROR:x11_software_bitmap_presenter.cc(144)] XGetWindowAttributes failed for window 220200964
[844745:0729/163751.981051:ERROR:network_service_instance_impl.cc(599)] Network service crashed, restarting service.
[844745:0729/163752.145460:ERROR:network_service_instance_impl.cc(599)] Network service crashed, restarting service.
[844745:0729/163752.356035:ERROR:network_service_instance_impl.cc(599)] Network service crashed, restarting service.
[844745:0729/163752.503198:ERROR:network_service_instance_impl.cc(599)] Network service crashed, restarting service.
`

**Comment by marcelo-js** [comment]
Seems to be working great now as of 3.25.102. Thanks for the fix!

**Comment by elmo667** [comment]
> Seems to be working great now as of 3.25.102. Thanks for the fix!

yes, can confirm

**Comment by stmasta** [comment]
I still have an app crash when viewing 1 thing, version 104, please help
